cnd/migrations/2020-05-06-105000_created-swap-tables/up.sql

```diff
@@ -0,0 +1,64 @@
+-- Your SQL goes here
+
+CREATE TABLE swaps
+(
+    id INTEGER			NOT NULL PRIMARY KEY,
+    local_swap_id UNIQUE	NOT NULL,
+    role			NOT NULL,
+    counterparty_peer_id	NOT NULL
+);
+
+CREATE TABLE secret_hashes
+(
+    id INTEGER			NOT NULL PRIMARY KEY,
+    swap_id INTEGER		NOT NULL,
+    secret_hash		NOT NULL,
+    FOREIGN KEY (swap_id)	REFERENCES swaps(id)
+);
+
+CREATE TABLE shared_swap_ids
+(
+    id INTEGER			NOT NULL PRIMARY KEY,
+    swap_id INTEGER		NOT NULL,
+    shared_swap_id		NOT NULL,
+    FOREIGN KEY (swap_id)	REFERENCES swaps(id)
+);
+
+CREATE TABLE address_hints
+(
+    id INTEGER			NOT NULL PRIMARY KEY,
+    swap_id INTEGER		NOT NULL,
+    address_hint		NOT NULL,
+    FOREIGN KEY(swap_id)	REFERENCES swaps(id)
+);
+
+CREATE TABLE herc20_swap_details
```

Comment:
Not a fan of the `swap_detail` terminology :)

```suggestion
CREATE TABLE herc20_protocol
```

Reply:
This should have been argued on the spike.  Happy to debate/improve it at the 99% review though :)

Reply:
> This should have been argued on the spike. Happy to debate/improve it at the 99% review though :)

I disagree, these kind of comments/discussions would have just delayed the spike from being merged.
Spikes shouldn't be exact instructions but just solve the hard problems.

As you said, names can always be changed so I've kept my feedback on this until we actually implement it.

Reply:
Cool, wish I realized that before starting; I would have changed a bunch of names :) Will keep this in mind for next time I code off of a spike.

Reply:
How about just plain `herc20`, since this is the main table for her20 info does adding anything extra to the name add any meaning?

Reply:
Fine with me :)